tributors/main/parsers/mailmap.py

```diff
@@ -38,15 +38,19 @@ def load_data(self):
                 sys.exit("%s does not exist" % self.filename)
 
             for line in read_file(self.filename):
-                name, email = line.split("<")
-                email = email.strip().rstrip(">")
-                self.data[email] = {"name": name.strip()}
+                # mailmap line can have more than one entry, split by right >
+                for entry in line.strip().split(">"):
+                    if not entry:
+                        continue
+                    name, email = entry.split("<")
+                    email = email.strip()
+                    self.data[email] = {"name": name.strip()}
```

Comment:
I believe it should be the `name` of the first entry which is used for all emails you can find later in the line. That is the point of such mailmap entries - to map into another name

Reply:
For the example you gave me, there were pairs of names and emails (so more than one name per line). You are saying if there isn’t a second name to use the first?

Reply:
Yes
You can run git shortlog -sn to see the effect of the second names collapsing into the record for the first in line

Reply:
okay gave it a shot (but didn't test yet) - take a look at changes!